website/docs/reference/widgets/progress.md

```diff
@@ -1,39 +1,121 @@
-| **Infinite loading** | Control’s the widget to be in an infinite loading state, which is useful if the progress values are not determinable. This can be switched on for queries or API calls that take time to return data.                                            |
-| **Type**             | The progress bar can be of linear or circular type.                                                                                                                                                                                              |
-| **Progress**         | Percentage of progress to be indicated to the user.                                                                                                                                                                                              |
-| **Number of steps**  | Progress bar can be broken down into multiple parts called steps, each step contains a fixed percentage of progress. Number of steps can be configured to break down the progress bar for better communication. Only supports positive integers. |
-| **Show Result**      | Controls widget’s ability to show the current evaluated percentage as in number along with the progress.                                                                                                                                         |
-| **Visible**          | Control widget's visibility on the page. When turned off, the widget isn't visible when the app is published.                                                                                                                             |
+<dd>
 
-### Binding properties
+Controls the visibility of the widget. If you turn off this property, the widget would not be visible in View Mode. Additionally, you can use JavaScript by clicking on **JS** next to the **Visible** property to conditionally control the widget's visibility.
 
-These properties allow you to bind your Progress bar widget with any other widget in queries or JS objects. The following table lists all the binding properties.
+For example, if you want to make the widget visible only when the user selects "Yes" from a Select widget, you can use the following JavaScript expression: 
+```js
+{{Select1.selectedOptionValue === "Yes"}}
+```
 
-| Property     | Description                                                      | Code Snippet                          |
-| ------------ | ---------------------------------------------------------------- | ------------------------------------- |
-| **progress** | It shows the current progress of the progress bar in percentage. | `{{<progress-widget_name>.progress}}` |
-| **visible**  | Visibility of the progress bar widget.                           | `{{<progress-widget_name>.visible}}`  |
+</dd>
 
-### Styles
+#### Show result `boolean`
 
+<dd>
+
+Allows you to enable or disable the display of the current evaluated percentage as a number along with the progress in the widget.
+
```

Comment:
You missed the **Counterclockwise** property for the Circular type progress bar

Reply:
updated